### PR TITLE
Add post on extension ecosystem jobs to be done

### PR DESCRIPTION
**Pull request by theory**

### CODE REVIEW

File: landing/src/content/blog/2024-02-12-extension-ecosystem-jobs/index.md [hunk 1]
```diff
@@ -0,0 +1,364 @@
...
+have added a limited number of extensions to their registries, such as
+[apt.postgresql.org], [yum.postgresql.org], and the like, but neither PGXN nor
+the [pgxn client][pgxnclient] provide integration into such tooling.
+
+As a result of these challenges, a number of new entrants have emerged in recent
+months, all aiming to be the source of record for extensions and a focus on
+discovery and ease of installation. These include [dbdev], [trunk][pgt.dev],
+[pgxman], [pgpm]. They too exhibit limitations, such as minimal OS and platform
+availability and, notably, the manual process to add extensions, further
+constraining inclusion. None has even reached the number provided by PGXN.
```
Comment (theory) [inline landing/src/content/blog/2024-02-12-extension-ecosystem-jobs/index.md:35]: Could potentially cut these three graphs.

File: landing/src/content/blog/2024-02-12-extension-ecosystem-jobs/index.md [hunk 2]
```diff
@@ -0,0 +1,364 @@
...
+### Authoritative
+
+**Job: Be the official, canonical source of record for all Postgres extensions**
+
+Developers need to know the one place to reliably release and find all
+publicly-available Postgres extensions. It can't be a subset, but must be
+comprehensive, the assumed default recognized and used by the community and
+included in the Postgres documentation.
+
+Exemplars:
```
Comment (theory) [inline landing/src/content/blog/2024-02-12-extension-ecosystem-jobs/index.md:77]: @samay-sharma I recall you suggested doing without the exemplars, but I think they're kind of useful. Could see it going either way, so kept their link references close by so they'd be easier to delete if we did away with the exemplars.

File: landing/src/content/blog/2024-02-12-extension-ecosystem-jobs/index.md [hunk 3]
```diff
@@ -0,0 +1,363 @@
+---
+slug: pgxn-ecosystem-jobs
+title: "Postgres Extension Ecosystem: Jobs to be Done"
+authors: [theory]
+tags: [postgres, extensions, ecosystem, packaging, distribution, architecture, jobs to be done]
+# image:
```
Comment (DarrenBaldwin07) [inline landing/src/content/blog/2024-02-12-extension-ecosystem-jobs/index.md:6]: Be sure to ad an  `image` before merging :)

**Review by DarrenBaldwin07 — APPROVED** [review]

File: landing/src/content/blog/2024-02-12-extension-ecosystem-jobs/index.md (hunk 3 above)
Comment (theory) [reply landing/src/content/blog/2024-02-12-extension-ecosystem-jobs/index.md:6]: Okay, what image?

File: landing/src/content/blog/2024-02-12-extension-ecosystem-jobs/index.md [hunk 4]
```diff
@@ -0,0 +1,363 @@
+---
+slug: pgxn-ecosystem-jobs
+title: "Postgres Extension Ecosystem: Jobs to be Done"
```
Comment (samay-sharma) [inline landing/src/content/blog/2024-02-12-extension-ecosystem-jobs/index.md:3]: I think the title can be more attractive :).

Some ideas / thoughts:
* Defining the future of Postgres Extension Ecosystem.
* Thoughts on the ideal Postgres Extension Ecosystem
* Jobs of the ideal Postgres Extension Ecosystem.
* Defining what the future of the Postgres Extension Ecosystem should be like

Can you think more about some of the ideas and propose something? Things to emphasize from my point of view.
* We're trying to define what an ideal Postgres extensions ecosystem would look like.
* This post focusses on the what and not the how.

File: landing/src/content/blog/2024-02-12-extension-ecosystem-jobs/index.md [hunk 5]
```diff
@@ -0,0 +1,363 @@
...
+
+Furthermore, PGXN distributes source code packages, requiring developers to
+download, compile, install, and test them. Binary packagers and cloud providers
+have added a limited number of extensions to their registries, such as
+[apt.postgresql.org], [yum.postgresql.org], and the like, but neither PGXN nor
+the [pgxn client][pgxnclient] provide integration into such tooling.
+
+As a result of these challenges, a number of new entrants have emerged in recent
+months, all aiming to be the source of record for extensions and a focus on
+discovery and ease of installation. These include [dbdev], [trunk][pgt.dev],
```
Comment (samay-sharma) [inline landing/src/content/blog/2024-02-12-extension-ecosystem-jobs/index.md:32]: Could we add a little extra emphasis on trunk, just a line or so?

These include dbdev, pgxman, pgpm and Tembo's own trunk. While projects like trunk fulfill use cases (eg. allowing Tembo Cloud to support 200+ extensions), they too exhibit limitations as a community solution...

File: landing/src/content/blog/2024-02-12-extension-ecosystem-jobs/index.md [hunk 6]
```diff
@@ -0,0 +1,363 @@
...
+availability and, notably, the manual process to add extensions, further
+constraining inclusion. None has even reached the number provided by PGXN.
+
+These challenges and the interest and energy put into exploring new solutions
+make clear that the time has come to revisit the whole idea of the PostgreSQL
+extension ecosystem: to work though the jobs to be done, specify the tools to do
+those jobs, and outline a plan for the broader Postgres community to design and
+build them.
+
+Future posts will dream up the tools nad make the plan; today we begin with the
```
Comment (samay-sharma) [inline landing/src/content/blog/2024-02-12-extension-ecosystem-jobs/index.md:43]: typo

File: landing/src/content/blog/2024-02-12-extension-ecosystem-jobs/index.md (hunk 2 above)
Comment (samay-sharma) [reply landing/src/content/blog/2024-02-12-extension-ecosystem-jobs/index.md:77]: I think the exemplars are good.

**Review by samay-sharma — COMMENTED** [review]
Looks great. I think my main feedback is a better title and a little bit of an update in the intro.

File: landing/src/content/blog/2024-02-12-extension-ecosystem-jobs/index.md (hunk 4 above)
Comment (theory) [reply landing/src/content/blog/2024-02-12-extension-ecosystem-jobs/index.md:3]: Kind of like having "Jobs to be Done" in the title, as it's an entrepreneurial term of art. Perhaps "The Jobs to be Done by the Ideal Postgres Extension Ecosystem".

File: landing/src/content/blog/2024-02-12-extension-ecosystem-jobs/index.md (hunk 6 above)
Comment (theory) [reply landing/src/content/blog/2024-02-12-extension-ecosystem-jobs/index.md:43]: OMG that typo 😂

File: landing/src/content/blog/2024-02-12-extension-ecosystem-jobs/index.md (hunk 3 above)
Comment (theory) [reply landing/src/content/blog/2024-02-12-extension-ecosystem-jobs/index.md:6]: I don't, no.

File: landing/src/content/blog/2024-02-12-extension-ecosystem-jobs/index.md (hunk 5 above)
Comment (theory) [reply landing/src/content/blog/2024-02-12-extension-ecosystem-jobs/index.md]: Tweaked in 8e00cab

File: landing/src/content/blog/2024-02-12-extension-ecosystem-jobs/index.md (hunk 4 above)
Comment (theory) [reply landing/src/content/blog/2024-02-12-extension-ecosystem-jobs/index.md]: Pushed with that title in d195f19.

**Comment by samay-sharma** [comment]
Made an attempt to shorten the intro. I think we should be good to ship after adding the image.

**Review by samay-sharma — APPROVED** [review]

**Pull request merged**